library/src/main/java/com/novoda/downloadmanager/MigrationNotificationChannelCreator.java

```diff
@@ -0,0 +1,39 @@
+package com.novoda.downloadmanager;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.content.res.Resources;
+import android.os.Build;
+
+class MigrationNotificationChannelCreator implements NotificationChannelCreator {
```

Comment:
A Migration will likely require a separate channel, some users / client applications may not want to surface notifications to users.